.eslintrc.js

```diff
@@ -162,14 +161,13 @@ const strictModeRules = {
 };
 
 const variablesRules = {
-	'init-declarations': TRANSITION_WARNING,
 	'no-catch-shadow': ERROR,
 	'no-label-var': ERROR,
 	'no-shadow': ERROR,
 	'no-shadow-restricted-names': ERROR,
 	'no-undef-init': ERROR,
 	'no-undefined': ERROR,
-	'no-unused-vars': ERROR,
```

Comment:
Error seems too severe for something that's not really an issue except for code cleanliness.